modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To query the cluster resource metrics API.
+* To monitor or manage alerts and silences.
+* To access Alertmanager component.
+
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
+
+|===
+|Name |Description
+
+|`cluster-monitoring-metrics-api` |Users with this role have the ability to query the cluster resource metrics API for core platform monitoring.
```

Comment:
should we refer to https://docs.openshift.com/container-platform/4.17/observability/monitoring/accessing-third-party-monitoring-apis.html#openshift-monitoringthanos-querier-2 and https://docs.openshift.com/container-platform/4.17/observability/monitoring/accessing-third-party-monitoring-apis.html#openshift-monitoringthanos-querier?
After checking the code, it also allows access to user-defined Prometheus and Thanos Ruler API endpoints.
```suggestion
|`cluster-monitoring-metrics-api` |Users with this role have the ability to query Thanos Querier web server on port 9091.
```

Reply:
As the current docs only mention `cluster-monitoring-view` as a required cluster role to be able to access this port, should I also add to [these docs](https://docs.openshift.com/container-platform/4.17/observability/monitoring/accessing-third-party-monitoring-apis.html#openshift-monitoringthanos-querier-2) that the access is also available with the `cluster-monitoring-metrics-api` role?

Reply:
Yes we'd need to update these docs too. The source is [there](https://github.com/openshift/cluster-monitoring-operator/blob/master/Documentation/resources.adoc) so I can fix it here first.

Reply:
Oh great, thanks! I just noticed that we generate that part of docs automatically, so it would be great if you fixed it there :+1: